public/index.js

```diff
@@ -1,66 +0,0 @@
-let restList = document.getElementById('restaurant')
-let markerArray = []
-
-//puts map on page 
-let myMap = L.map('map').setView([44.47945238700768, -73.2185983657837], 15)
-myMap.on('load', loadMapData());
-
-L.tileLayer('https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png', {
-    maxZoom: 17,
-    attribution: 'Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
-}).addTo(myMap)
-
-
-function loadMapData() {
-    //first fetch json with coordinates
-    fetch('https://yelpingtonapi.herokuapp.com/api/restaurants')
-        .then((res) => {
-            return res.json()
-        })
-        .then((obj) => {
-
-            obj.forEach((restaurant) => {
-                //fetching json with other restaurant info 
-                //"id" "name" "address" "coords" "category" "price" "phone" "website" "hours" "notes"  
-                fetch(restaurant.infoUrl)
-                    .then((res) => {
-                        return res.json()
-                    }).then((restMap) => {
-                        let restLink = `<div><a href = "/restaurant.html?restaurant=${restMap.id}"> ${restMap.name}</a></div>`
-                        restList.innerHTML += restLink //build the list of restaurants w/links
-                        placeMarker(restMap.coords, restLink) //put links in popup pins
-
-
-                    }).catch(err => console.log(err))
-
-            });
-
-        }).finally(() => {
-            //center map in middle of markers, cool map movement feature!
-            //can be commented out to use fixed set view coordinates but this allows for adjustments if new restaurants are added and brings a fun movement feature upon app opening!
-            setTimeout(function () {
-
```

Comment:
Wow, Jena!  This is extremely well thought out, I'm really impressed.  You're implementing parts of the fetch API I don't believe we went over in class, with the `finally` method, as well some Leaflet stuff.  It's clear you did your research and applied things outside of what was shown, which is a huge deal in programming. Finding other ways around the problem, and consulting the documentation to do it *cleanly* is essential.  Great job.